packages/pie-models/README.md

```diff
@@ -0,0 +1,77 @@
+
+Typescript types & JSON Schemas for PIE Elments.
+
+This project contains typescript interfaces for models used with PIE components.
+It also generates JSON schema documentation for the models. 
+
+
+## Usage
+
+`npm install`
+
+This will install the typescript lib and also provide the json schemas in `dist/schemas`.
+
+
+### Typescript
+
+Using the types will provide IDE auto-complete and type-checking on model objects:
+
+`npm install @pie-element/pie-models`
+
+```ts
+import {MultipleChoiceConfig} from `@pie-element/pie-models`;
```

Comment:
Sorry - I missed this pr .. 
wouldn't it be better if we have: 
```javascript
import { mc } from '@pie-lib/multiple-choice';

let myConfig = types.Configure;
```

Aka put the types in the relevant package. These will be imported anyway so you won't have to add an extra import just for the type defs?

The type defs should not be part of the git repo, but are generated when  we run `yarn build` or `yarn release`.